autoscan is getting a warning from perl on this RE.  Fix it.
Should pass this upstream.


--- autoconf-2.69/bin/autoscan.in_orig	2016-03-15 17:17:51.090635380 -0700
+++ autoconf-2.69/bin/autoscan.in	2016-03-15 17:18:10.667432500 -0700
@@ -358,7 +358,7 @@
     {
       # Strip out comments and variable references.
       s/#.*//;
-      s/\${[^\}]*}//g;
+      s/\$\{[^\}]*}//g;
       s/@[^@]*@//g;
 
       # Tokens in the code.
